cswinrt/cswinrt.vcxproj

```diff
@@ -7,6 +7,7 @@
     <Keyword>Win32Proj</Keyword>
     <ProjectName>cswinrt</ProjectName>
     <RootNamespace>cswinrt</RootNamespace>
+    <Nullable>enable</Nullable>

```

Comment:
This is a C++ project, I assume nullable doesn't have any effect as it is C#?

Reply:
The idea was to not miss strings directory (which is C#)